Header header
int32 robot_id
int32 state
string path
string probs
